code/manager/ui/src/components/layout/desktop.stories.tsx

```diff
@@ -15,18 +15,12 @@ export default {
   decorators: [
     ((StoryFn, c) => {
       const mocked = true;
-      const height = 900;
-      const width = 1200;
 
-      if (isChromatic) {
+      if (isChromatic()) {
         store.local.set(`storybook-layout`, {});
       }
 
-      const props = {
-        height,
-        width,
-        ...(mocked ? mockProps : realProps),
-      };
+      const props = mocked ? mockProps : realProps;
```

Comment:
TS complained that `height` and `width` would always get overwritten, which is true because they are both present in `mockProps` and `realProps`